components/event/EventCard.js

```diff
@@ -75,11 +76,11 @@ export default function EventCard({ manage, event, usernames }) {
                     </Link>
                   )}
                 </div>
-                {event.userStatus && (
+                {event.userStatus == config.userStatus && (
                   <div className="text-primary-medium-low dark:text-primary-low-medium italic hidden lg:block">
                     {event.userStatus}
-                    {event.userStatus == "speaking" && " at "} this event
-                    {event.userStatus == "speaking" && event?.speakingTopic && (
+                    {event.userStatus == config.userStatus && " at "} this event
+                    {event.userStatus == config.userStatus && event?.speakingTopic && (
```

Comment:
Based on new config updation i.e : 
 ```
 "layouts": [
    "classic",
    "inline",
    "column-2"
  ],
  "events":{
    "userStatus":["Attending","Speaking"]
  },
```
Here is the change, what do you think Eddie?
```
             {event.userStatus == config.events.userStatus[1] && (
                  <div className="text-primary-medium-low dark:text-primary-low-medium italic hidden lg:block">
                    {event.userStatus}
                    {event.userStatus == config.events.userStatus[1] && " at "} this event
                    {event.userStatus == config.events.userStatus[1] && event?.speakingTopic && (
                      <>
                        {" "}
                        on <b>{event.speakingTopic}</b>
                      </>
                    )}
                  </div>
```